packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -124,6 +125,28 @@
       color: $inverse-link;
     }
   }
+
+  .#{$prefix}--aspect- {
+    &2x1 {
+      @include ratio-base(2, 1);
+    }
+
+    &16x9 {
+      @include ratio-base(16, 9);
+    }
+
+    &3x2 {
+      @include ratio-base(3, 2);
+    }
+
+    &4x3 {
+      @include ratio-base(4, 3);
+    }
+
+    &1x1 {
+      @include ratio-base(1, 1);
+    }
+  }
```

Comment:
Just checking, it looks like we're adding these as options to the card itself. We're good with this? 
cc: @photodow @kennylam @annawen1 @andysherman2121

Reply:
I don't see a problem with adding those options